sample-plugin/pom.xml

```diff
@@ -429,6 +429,11 @@
       <artifactId>docker-java-api</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.jenkins-ci.plugins</groupId>
+      <artifactId>docker-workflow</artifactId>
```

Comment:
Thanks.  I have removed docker-commons and confirmed that sample plugin tests pass locally.  That has been pushed as f0bb487d483c957291d533141fbd7684f81d0c75

Since `docker-workflow` depends on several other plugins, should I remove those other dependencies from the sample plugin as well?

## `docker-workflow` Dependencies

* [Pipeline: Basic Steps ≥ 1017.vb_45b_302f0cea_](https://plugins.jenkins.io/workflow-basic-steps/)
* [Pipeline: Groovy ≥ 3659.v582dc37621d8](https://plugins.jenkins.io/workflow-cps/)
* [Pipeline: Nodes and Processes ≥ 1246.v5524618ea_097](https://plugins.jenkins.io/workflow-durable-task-step/)
* [Pipeline: Step API ≥ 639.v6eca_cd8c04a_a_](https://plugins.jenkins.io/workflow-step-api/)
* [Folders ≥ 6.758.vfd75d09eea_a_1](https://plugins.jenkins.io/cloudbees-folder/)
* [Docker Commons ≥ 419.v8e3cd84ef49c](https://plugins.jenkins.io/docker-commons/)
* [Pipeline: Declarative ≥ 2.2131.vb_9788088fdb_5](https://plugins.jenkins.io/pipeline-model-definition/)

Reply:
Sure, you could.

Reply:
Thanks.  If this run fails, I'll add those changes to the next run.  If it succeeds and merges, I'll submit them as a separate pull request.

Reply:
https://github.com/jenkinsci/bom/pull/2470 is the pull request that removes several transitive dependencies from the pom of the sample plugin.